spec/controllers/external_authors_controller_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ExternalAuthorsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { FactoryGirl.create(:user) }
+  let(:invitation) { FactoryGirl.create(:invitation, external_author: external_author) }
+  let(:external_author) { FactoryGirl.create(:external_author) }
+
+  before(:each) do
+    fake_login_known_user(user)
+  end
+
+  describe "GET #get_external_author_from_invitation" do
```

Comment:
In the previous version, these were testing what happened while logged out, but now they're testing the behavior while logged in. I'm not sure if that's important, but it makes me wonder if we should be testing both logged in and out behavior.